packages/ui5-library-writer/templates/common/src/baselibrary/Example.gen.d.ts

```diff
@@ -0,0 +1,33 @@
+import { ExampleColor } from "./library";
```

Comment:
Is this file generated with https://github.com/SAP/ui5-typescript/tree/main/packages/dts-generator?
If yes, shouldn't we have a script in the `package.json` to regenerate it if the user change `Example.js`?

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/1008/commits/e7fe06c282cfc61b92a2c775d630831b23774710#diff-ca37b8e6576d8edc6acc4a50e6ccb412db91e3b980a3ff3620ade56f455abfa4R29